src/jit/jitconfig.h

```diff
@@ -21,11 +21,11 @@ class JitConfigValues
             MethodName* m_next;
             int         m_methodNameStart;
             int         m_methodNameLen;
-            bool        m_methodNameWildcardAtEnd;
             int         m_classNameStart;
             int         m_classNameLen;
-            bool        m_classNameWildcardAtEnd;
             int         m_numArgs;
+            bool        m_methodNameWildcardAtEnd;
```

Comment:
You should add a comment here that these are here to improve packing. Note that improving packing in this struct is not very important...